src/configure/configure.ts

```diff
@@ -154,6 +157,23 @@ class Orchestrator {
         }
     }
 
+    private async getGithubPatToken(): Promise<void> {
+        const redirectHelper = new RedirectLinkHelper();
+        await redirectHelper.loadAll();
+        if (this.inputs.sourceRepository.repositoryProvider === "github" || redirectHelper.repoAnalysisServiceFramework === ServiceFramework.Moda) {
+            this.inputs.githubPATToken = await this.controlProvider.showInputBox(constants.GitHubPat, {
+                placeHolder: Messages.enterGitHubPat,
+                prompt: Messages.githubPatTokenHelpMessage,
```

Comment:
the prompt message is different in two cases as one requires additional scopes. please handle this.

Reply:
The PAT token can by default be used to authenticate to github API. Additional scopes needed are repo read access for github workflow, which is already being informed to the user

Reply:
Don't we need additional scope for pushing workflow file in case of github workflow?

Reply:
No, read/write access is what is needed for secrets api, pushing workflow file etc.